joy_song/about2.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang = "en-US">
+    <head>
+        <!--Ideas:
```

Comment:
When pushing code in general and especially when making a PR, make sure you remove all commented code. Comments are good when they clarify something unclear about the code, but you can keep comments such as future ideas somewhere else.